products/mariadb.md

```diff
@@ -14,52 +14,62 @@ command: mysqld --version
 eolColumn: Support Status
 sortReleasesBy: 'releaseCycle'
 releases:
-  - releaseCycle: "10.7"
-    release: 2022-02-14
+-   releaseCycle: "10.7"
+    release: 2021-11-05
     eol: 2023-02-14
     latest: "10.7.3"
     latestShortHand: "1073"
-  - releaseCycle: "10.6"
-    release: 2021-07-06
+    latestReleaseDate: 2022-02-10
```

Comment:
https://mariadb.com/kb/en/mariadb-1073-release-notes/ says 2022-02-12

Reply:
Corresponding Tag: https://api.github.com/repos/MariaDB/server/git/tags/2aa8e7d6db3bcb0019d23882d29251d8969fbe61

https://github.com/MariaDB/server/releases/tag/mariadb-10.7.3

Date shows up as `2022-02-12T18:38:44Z`, so I think this is a timezone issue.

Reply:
So the issue was in how the code was parsing annotated tags. The time was getting picked up from the commit instead of annotation. 

- Commit: https://github.com/MariaDB/server/commit/65f602310c4ea086488bfaa50f36952162541a8c (10th Feb)
- Tag Annotation: https://github.com/MariaDB/server/releases/tag/mariadb-10.7.3 (12th Feb)

This is now resolved.